host-bmc/host_pdr_handler.hpp

```diff
@@ -452,6 +471,13 @@ class HostPDRHandler
     PDRList stateSensorPDRs;
     PDRList fruRecordSetPDRs{};
 
+    /** @brief Entity Auxiliary Name PDRs list */
+    PDRList entityAuxNamesPDRs{};
+
+    /** @brief Entity Auxiliary Names list */
+    std::vector<std::shared_ptr<pldm::hostbmc::utils::EntityAuxiliaryNames>>
```

Comment:
I think changing the datatype of "entityAuxiliaryNamesList" to map will simplify things and avoid unnecessary calculations. Do you think otherwise?

Reply:
It would be ideally same as I need to compare against each of the PDR entries to entity key parameters and there is no special calculation used here.